Patch greedy dot star to fix mismatched pkg-config file version.
Fixed upstream with: https://github.com/jmcnamara/libxlsxwriter/commit/12afb351db634ed9452eb2f38d68883a9538b8ef
https://bugs.gentoo.org/859745

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -202,9 +202,9 @@
 # Configure pkg-config
 file(READ "include/xlsxwriter.h" ver)
 
-string(REGEX MATCH "LXW_VERSION .*\"\(.*\)\".*" _ ${ver})
+string(REGEX MATCH "LXW_VERSION \"([^\"]+)\"" _ ${ver})
 set(VERSION ${CMAKE_MATCH_1})
-string(REGEX MATCH "LXW_SOVERSION .*\"\(.*\)\".*" _ ${ver})
+string(REGEX MATCH "LXW_SOVERSION \"([^\"]+)\"" _ ${ver})
 set(SOVERSION ${CMAKE_MATCH_1})
 set(PREFIX ${CMAKE_INSTALL_PREFIX})
 
